clients/admin-ui/src/features/datastore-connections/DisableConnectionModal.tsx

```diff
@@ -38,15 +38,13 @@ const DisableConnectionModal: React.FC<DataConnectionProps> = ({
 
   const handleDisableConnection = async () => {
     const shouldDisable = !disabled;
-    return patchConnection({
+    patchConnection({
       key: connection_key,
       name,
       disabled: shouldDisable,
       access: access_type,
       connection_type,
     })
-      .unwrap()
```

Comment:
Does the modal still close once the request finishes?

Reply:
Yup, it still closes automatically once the request completes

Reply:
That's great. So was the issue was calling the `onClose` function after the modal had closed?